modin/pandas/series.py

```diff
@@ -1425,9 +1425,13 @@ def unstack(self, level=-1, fill_value=None):  # noqa: PR01, RT01, D200
         """
         from .dataframe import DataFrame
 
-        result = DataFrame(
-            query_compiler=self._query_compiler.unstack(level, fill_value)
-        )
+        # We can't unstack a Series object, if we don't have a MultiIndex.
+        if len(self.index.names) > 1:
+            result = DataFrame(
+                query_compiler=self._query_compiler.unstack(level, fill_value)
+            )
+        else:
+            raise ValueError(f"index must be a MultiIndex to unstack, {type{obj.index}} was passed")
```

Comment:
Thinking about adding a `pandas_name` attribute to our ponder Indexes, and using that so we can get something kind of similar to what users would expect for this error. Can always revert it back to not showing type as is done in Modin-public.